e2e/e2e_test.go

```diff
@@ -282,7 +282,7 @@ var _ = Describe("Whereabouts functionality", func() {
 					Expect(err).NotTo(HaveOccurred())
 					Expect(podList.Items).NotTo(BeEmpty())
 
-					ipPool, err := k8sIPAM.GetIPPool(context.Background(), ipPoolName)
+					ipPool, err := k8sIPAM.GetIPPool(context.Background(), ipPoolCIDR, "")
```

Comment:
please use a constant instead of the magic string `""`. Name it something like `uniquePerCIDRPoolName` or something like that. 

I would welcome any other name ... That one was not good :)

Reply:
✅